### PR TITLE
Add sphinxcontrib-programoutput 0.11

### PR DESCRIPTION
**Pull request by holgern**
https://pypi.python.org/pypi/sphinxcontrib-programoutput

### CODE REVIEW

File: recipes/sphinxcontrib-programoutput/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
```
Comment (CJ-Wright) [inline recipes/sphinxcontrib-programoutput/meta.yaml]: Is this a build requirement?

File: recipes/sphinxcontrib-programoutput/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+about:
+  home: https://bitbucket.org/birkenfeld/sphinx-contrib
+  license: BSD 2-clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Sphinx extension to include program output'
+
+  description: |
+    A Sphinx extension to literally insert the output of arbitrary commands into documents, helping you to keep your command examples up to date.
```
Comment (CJ-Wright) [inline recipes/sphinxcontrib-programoutput/meta.yaml]: Please break this up over multiple lines

Comment (holgern) [reply recipes/sphinxcontrib-programoutput/meta.yaml]: Done

File: recipes/sphinxcontrib-programoutput/meta.yaml (hunk 1 above)
Comment (holgern) [reply recipes/sphinxcontrib-programoutput/meta.yaml]: I will try with Sphinx and try to remove the other ones.

**Pull request merged**